src/AngularPublic.js

```diff
@@ -265,5 +265,6 @@ function publishExternalAPI(angular) {
         $$cookieReader: $$CookieReaderProvider
       });
     }
-  ]);
+  ])
+  .info({ angularVersion: '"NG_VERSION_FULL"' });
```

Comment:
To me `angularVersion` seems more intuitive; otherwise I'd feel we're talking about the module version as Pete said.